components/core/core-database/src/main/java/org/eclipse/dirigible/components/database/DataSourceSystemConfig.java

```diff
@@ -45,10 +45,10 @@ public class DataSourceSystemConfig {
     public HikariDataSource getDataSource() {
         DataSourceProperties dataSourceProperties = new DataSourceProperties();
         dataSourceProperties.setName("SystemDB");
-        dataSourceProperties.setDriverClassName("org.h2.Driver");
-        dataSourceProperties.setUrl("jdbc:h2:file:./target/dirigible/h2/SystemDB");
-        dataSourceProperties.setUsername("sa");
-        dataSourceProperties.setPassword("");
+        dataSourceProperties.setDriverClassName(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_DRIVER", "org.h2.Driver"));
+        dataSourceProperties.setUrl(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_URL", "jdbc:h2:file:./target/dirigible/h2/SystemDB"));
+        dataSourceProperties.setUsername(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_USERNAME", "sa"));
+        dataSourceProperties.setPassword(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_PASSWORD", ""));
```

Comment:
you can add the constants [here](https://github.com/eclipse/dirigible/blob/master/modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java#L44) as well